examples/aobench/src/vector.rs

```diff
@@ -81,7 +81,7 @@ cfg_if! {
         pub fn ao<S: Scene>(scene: &mut S, nsubsamples: usize,
                             img: &mut ::Image) {
             unsafe {
-                if is_x86_feature_detected!("avx2") {
+                if is_x86_feature_detected!("avx2") && is_x86_feature_detected!("fma") {
                     ao_avx2(scene, nsubsamples, img);
                 } else if is_x86_feature_detected!("avx") {
```

Comment:
I think we should also have a branch here for AVX+FMA (and also one for AVX without FMA)